mod/04_settings/set/right/comment.rb

```diff
@@ -33,7 +33,7 @@
               %{
                 <div class="group-option">
                   #{ check_box_tag( "#{option.key}-perm-checkbox", option.name, checked, :class=>'perm-checkbox-button'  ) }
-                  <label>#{ link_to_page option.name, nil, :target=>'wagn_role' }</label>
+                  <label>#{ build_link option.name, nil, :target=>'wagn_role' }</label>
```

Comment:
Not surprising you didn't see it as it isn't that obvious.  I could have added that comment earlier.  All good.  Took me a bit to realize it was only handling class explicitly before.